apollo-normalized-cache/src/main/java/com/apollographql/apollo/cache/normalized/lru/EvictionPolicy.kt

```diff
@@ -0,0 +1,92 @@
+package com.apollographql.apollo.cache.normalized.lru
+
+import com.apollographql.apollo.api.internal.Optional
+import java.util.concurrent.TimeUnit
+
+/**
+ * Controls how long a [com.apollographql.apollo.cache.normalized.Record] will
+ * stay in a [LruNormalizedCache].
+ */
+class EvictionPolicy internal constructor(
+    private val maxSizeBytes: Optional<Long>,
```

Comment:
Can you remove usages of Optional and replace them with Kotlin Nullable type? Of course we need to be careful in places where these are exposed as public API. But I think it is still possible to remove as much as possible.